docs/DeveloperGuide.md

```diff
@@ -478,317 +478,330 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
+    <br>
+    Use case ends
+* 2b. Duplicate consultation Exists (Consultation date & time matches an existing consultation)
+    * 1b1. TAHub shows a duplicate error message.
+    <br>
+    Use case ends.
 
-* 1a. No existing consultations
-    * 1a1. System shows message that no future consultations are available
-    * 1a2. Use case ends
-
-* 2a. Invalid consultation selection
-    * 2a1. System shows error message
-    * 2a2. Use case resumes from step 1
-
-* 3a. Student does not exist in system
-    * 3a1. System shows error message that student cannot be found
-    * 3a2. Use case resumes from step 3
-
-* 3b. Student already in consultation
-    * 3b1. System shows error message that student is already registered
-    * 3b2. Use case resumes from step 3
 
-**<u>Use case: UC9 - Delete Consultation</u>**
+**<u>Use case: UC10 - Add Student to Consultation</u>**
 
-**MSS**
+**Precondition:** The Consultation exists in TAHub.
 
-1. TA lists existing consultations
-2. TA enters command to delete specific consultation(s) by index
-3. System removes the consultation(s)
-4. Success message shown with deleted consultation details
+**MSS:**
+1. User requests to add a specific student to the consultation by providing the necessary details (Consultation Index, Student Index, Student Name)
+2. TAHub validates the inputs.
+3. TAHub adds the student to the consultation.
+4. TAHub displays the updated consultation list.
+<br>
+Use case ends.
 
 **Extensions**
+* 2a. Invalid Consultation Index
+    * 2a1. TAHub shows an error message stating that the Consultation Index is invalid.
+    <br>
+    Use case ends.
+* 2b. Invalid Student Index
+  * 2b1. TAHub shows an error message stating that the Student Index is invalid.
+    <br>
+    Use case ends.
+* 2c. Student Name not Found in Student List.
+  * 2c1. TAHub shows an error message stating that the Student does not exist.
+    <br>
+    Use case ends.
+* 2d. Student is already in consultation
+  * 2d1. TAHub shows an error message stating that the Student is already in the consultation.
+    <br>
+    Use case ends.
+
+
+**<u>Use case: UC11 - Remove Student from Consultation</u>**
+
+**Precondition:** The Consultation exists in TAHub.
 
-* 1a. No existing consultations
-    * 1a1. System shows message that no consultations are available
-    * 1a2. Use case ends
-
-* 2a. Invalid consultation index
-    * 2a1. System shows error message listing invalid indices
-    * 2a2. Use case resumes from step 2
-
-**<u>Use case: UC10 - Remove Students from Consultation</u>**
-
-**MSS**
-
-1. TA lists existing consultations
-2. TA selects consultation by index
-3. TA specifies students to remove by name
-4. System validates student existence in consultation
-5. System removes specified students from consultation 
-6. Success message shown
+**MSS:**
+1. User requests to remove a specific student from the consultation by providing the necessary details (Consultation Index, Student Index, Student Name)
```

Comment:
Correct me if I'm wrong but I don't think `removefromconsult` allows deleting a student by index, only their full name.

Reply:
You're correct. Will Modify accordingly.